packages/core/stories/tokens/colors/Colors.stories.mdx

```diff
@@ -1,98 +1,72 @@
 import { Meta } from '@storybook/addon-docs/blocks';
 import generateTitle from '../../util/generateTitle';
 import { ColorSwatch, GradientSpectrum } from './ColorDisplay';
-import { themeColors, grayScaleColors, utilityColors, primaryColors, primaryAltColors, highLightColors } from './colors.json';
+import { themeColors, grayScaleColors, utilityColors, hierarchicalColors} from './colors.json';
 
 <Meta title={generateTitle('Colors')} />
 
-# Colors
-Color is a key component of a brand. A well defined suite of colors can effeciently convey the brand and empower visual consistency. Apply colors systemematically across can help users set expections of the meaning and hierarchies of the perceived information.
+# Color
+
+We use color to establish brand consistency across all Massachusetts digital experiences, but also to convey semantic meaning—for example in warnings and alerts—as well as to differentiate categories of information.
 
 
 ## Brand Colors
-Mayflower brand colors are carefully selected to meet the brand pillars and adhere to specific accessibility standards.
-We encourage that all Commonwealth agencies use the same
-suite of colors in order to cement a cohesive image of the Commonwealth
-brand on all public-facing assets and documents.
-Organizations that have their own visual identity are invited to extend this color palette with additional colors that reflect their unique brand in conjunction with the shared Commonwealth identity.
+
+The Mayflower brand colors were selected to meet our brand and accessibility standards. All Commonwealth agencies should use these colors—along with other brand imagery—to signal to users that they are seeing official Commonwealth content. Organizations with their own visual identities may extend this color palette with their unique brand colors, but we encourage you to make an effort to harmonize your color palette with Mayflower as much as possible.
 
 <ul className="sg-colors">
   {
     themeColors.map((color, i) => <li key={`themeColors${i}`}><ColorSwatch {...color} variable={color.token} /></li>)
   }
 </ul>
 
-## Theme Colors
```

Comment:
I think we should name it "theme colors" to keep it consistent with the prop name for components, e.g. https://mayflower.digital.mass.gov/react/index.html?path=/story/atoms-buttons-button--button-example